src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -82,10 +98,23 @@
       <_CloneRepoLines Include="popd" />
     </ItemGroup>
 
+    <!-- Unix script -->
+    <ItemGroup Condition="'$(TargetsUnix)'=='true'">
+      <_CloneRepoLines Include="#!/usr/bin/env bash" />
```

Comment:
Oh I didn't know that, this is good to know. I basically followed this pattern because it is the way we have done it across our repos, for example [this targets file in buildtools](https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets#L63-L113). That said, I'm happy to change it if you think it's better to have it in a property.